packages/scripts/src/new/lib.ts

```diff
@@ -16,7 +16,7 @@ function writeJson(data: any) {
   const files = Object.keys(data)
 
   // write to each file
-  files.forEach(f => {
+  files.forEach((f) => {
```

Comment:
Let's revert all of the changes to this file. (Prettier might re-format it again, which is okay).
We don't have a github action build matrix anymore.